ext/dom/xml_common.h

```diff
@@ -76,8 +73,8 @@ PHP_DOM_EXPORT xmlNodePtr dom_object_get_node(dom_object *obj);
 	} \
 }
 
-#define DOM_RET_OBJ(obj, ret, domobject) \
-	*ret = php_dom_create_object(obj, return_value, domobject)
+#define DOM_RET_OBJ(obj, domobject) \
```

Comment:
is there still a point using it ?

Reply:
It's similar to how we have `RETVAL_OBJ(foo)` etc even though we could use `ZVAL_OBJ(return_value, foo)` instead. It's a bit less typing work :shrug:

Reply:
fair point. `DOM_PROP_NODE` is more justified.